.env.testing

```diff
@@ -2,7 +2,7 @@ APP_NAME="Talent Cloud"
 APP_ENV=testing
 APP_KEY=base64:lSoNWwEtogJWFNF7ryPiYyXeSdU4ked4NrhRDMvdUFE=
 APP_DEBUG=false
-APP_URL=https://talent.test
+APP_URL=http://localhost
```

Comment:
This might affect local development environments running unit tests

Reply:
Also not necessary, switched back.